presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -147,7 +147,8 @@ public Node visitUse(@NotNull SqlBaseParser.UseContext context)
     @Override
     public Node visitCreateTableAsSelect(@NotNull SqlBaseParser.CreateTableAsSelectContext context)
     {
-        return new CreateTableAsSelect(getQualifiedName(context.qualifiedName()), (Query) visit(context.query()));
+        boolean createTableWithoutData = context.query().withOrWithoutData() != null ? context.query().withOrWithoutData().NO() != null : false;
```

Comment:
will it be better to extract `context.query().withOrWithoutData()` into an expression? save some calls.